### PR TITLE
Feat/pm 888/tab overview desktop

### PR DESCRIPTION
**Pull request by mati-1**
[PM-888](https://playmakerpro.atlassian.net/browse/PM-888)

this branch only has tab overview changes please make cr and after correcting the code dont merge i want all tabs and top profile in this branch

- changed main layout with desktop version
- added navbar
- added all sections of overview tab
- corrected tables (i hope good)

### CODE REVIEW

File: components/common/Button/Button.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ const iconCva = cva('group-disabled:fill-darkAlpha-20', {
       large: 'w-7 h-7',
       medium: 'w-5 h-5',
       small: 'w-5 h-5',
-      extraSmall: 'w-4 h-4',
+      extraSmall: 'w-4 h-4 md:w-5 md:h-5',
```
Comment (dejwi) [inline components/common/Button/Button.tsx:45]: as we talked about it don't guess sizes

**Review by dejwi — CHANGES_REQUESTED** [review]
overall looks good💯

couple of things:

I don't really like this table change, I really wanted to avoid putting same height in multiple files, are you sure we can't do better ?
and you still rely on context rowHeight and expandedRowHeight on rows/main with paddingBottom (it's used in matches table) so that mixture is even worse

avatar border is cropped
image

can we make this chips not cropped on left ?
image

and this matches table border shouldn't be connected everywhere (figma image below)
image

File: components/common/Button/Button.tsx (hunk 1 above)
Comment (mati-1) [reply components/common/Button/Button.tsx:45]: i forgot to remove it

**Comment by mati-1** [comment]
i will be very grateful if you correct the tables to done your vision, it will take you much faster because you were building this component

**Review by dejwi — CHANGES_REQUESTED** [review]
I think this spacing is lil too big
image

add `desktopSize` to button and please fixup the matches table border

**Review by dejwi — CHANGES_REQUESTED** [review]
mostly looks nice, good job💪

pointer (I know this component is a shit show😅)
image

ŚŚŚŚ (in two tables) (+columns should wider but it'll be my task with tables)
image


header banner btn
(dev)
image

(figma)
image

and sections bottom paddings should be a little bigger

**Review by dejwi — APPROVED** [review]
- don't use AnimatePresence where it's not needed, it's so you can use exit animation but we didn't use it here (transfers, similar players) and I've looked into this because there were missing keys errors in browser console, look out for that as well bro!
- loaders still need to be corrected, tabs and whole page (more centered)
image

- and don't skip cr stuff !! I've mentioned in previous one that paddings in sections are off and you skipped that
- ^ pointer still needs to be fixed (picture with custom percentage passed - 90)
image

wel'll merge this as it is and changes above will come with separate pr

**Pull request merged**